miriway.cpp

```diff
@@ -197,8 +243,9 @@ struct ShellPids
 
                     if (auto it = shell_component_pids.find(pid); it != shell_component_pids.end())
                     {
-                        if (status)
+                        if (WIFEXITED(status) && WEXITSTATUS(status))
                         {
+                            mir::log_info("Process exited with status: %d", WEXITSTATUS(status));
```

Comment:
Who is this "info" for? It looks uninformative ("what process?") and unnecessary